entries/h/home.saxo.json

```
@@ -0,0 +1,17 @@
+{
+    "SaxoTraderGO": {
```

Saxo Bank or Saxo Group is probably a better name since the authentication system is for all Saxo platforms.
```suggestion
    "Saxo Bank": {
```